src/popup/stores/SendStore.ts

```diff
@@ -24,15 +31,31 @@ export default class SendStore {
   @observable public token?: QRCToken = INIT_VALUES.token;
   @observable public amount: string = INIT_VALUES.amount;
   @observable public maxAmount?: number = INIT_VALUES.maxAmount;
+  public transactionSpeeds: string[] = INIT_VALUES.transactionSpeeds;
+  @observable public transactionSpeed?: string = INIT_VALUES.transactionSpeed;
+  @observable public gasLimit?: string = INIT_VALUES.gasLimitRecommendedAmount;
+  @observable public gasPrice?: string = INIT_VALUES.gasPriceRecommendedAmount;
```

Comment:
suggest we store the `gasLimit` and `gasPrice` in its native number format and only change to string when needed. believe the `Textfield` should be able to handle it as a number.